### ISSUE TITLE
Please add KAFKAREST_HEAP_OPTS to documentation

### ISSUE THREAD

**Issue by jmarcos-cano** [confluentinc/cp-docker-images#297]
I did not find any reference for KAFKAREST_HEAP_OPTS nor 
 KAFKAREST_JVM_PERFORMANCE_OPTS in the documentation unless I went to 

${COMPONENT}-run-class in here: (`tail "${COMPONENT}"-start`)

`exec $(dirname $0)/kafka-rest-run-class io.confluent.kafkarest.KafkaRestMain "$@"`

**Comment by noderat** [comment]
Yeah, so this still isn't in the documentation. Five hours later I finally found it. Wow.

**Comment by funderbar** [comment]
it is 2019 now : )

**Comment by xdsxc** [comment]
Its almost 2020 now :)

**Comment by cojamac** [comment]
It's now 2020
